### ISSUE TITLE
Angular: add slot fallback

### ISSUE THREAD

**Issue by matschik** [matschik/component-party#29]
Directory: `/content/4-component-composition/4-slot-fallback`
https://github.com/matschik/component-party/tree/main/content/4-component-composition/4-slot-fallback

**Comment by nartc** [comment]
Angular does not have Slot Fallback. And the community has various different approaches. 

Here's one way you can do it

```ts
@Component({
    selector: 'app',
    template: `
        <funny-button>
            <ng-template #content>I got content!</ng-template>
        </funny-button>
        <funny-button></funny-button>
    `
})
export class AppComponent {}

@Component({
    selector: 'funny-button',
    template: `
        <button
	     style="background: rgba(0, 0, 0, 0.4); color: #fff; padding: 10px 20px; font-size: 30px; border: 2px solid #fff; margin: 8px; transform: scale(0.9); box-shadow: 4px 4px rgba(0, 0, 0, 0.4); transition: transform 0.2s cubic-bezier(0.34, 1.65, 0.88, 0.925) 0s; outline: 0;"
        >
            <ng-container *ngIf="content;else fallback" [ngTemplateOutlet]="content"></ng-container>
            <ng-template #fallback>
                <span>No content found</span>
            </ng-template>
        </button>
        <ng-content></ng-content>
    `
})
export class FunnyButtonComponent {
    @ContentChild('content', { read: TemplateRef }) content?: TemplateRef<unknown>;
}
```

**Comment by matschik** [comment]
Resolved in #41
